Extension/src/LanguageServer/ui.ts

```diff
@@ -426,12 +433,16 @@ export class LanguageStatusUI {
     private showConfigureIntelliSenseButton: boolean = false;
     private configureIntelliSenseTimeout?: NodeJS.Timeout;
 
-    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client): Promise<void> {
+    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client, configurationType?: ConfigurationType, sender?: string): Promise<void> {
+        if (configurationType !== undefined && sender !== undefined) {
+            const showButton: string = show ? 'true' : 'false';
+            telemetry.logLanguageServerEvent('showConfigureIntelliSenseButton', { configurationType, sender, showButton });
+        }
         if (!await telemetry.showStatusBarIntelliSenseButton() || client !== this.currentClient) {
             return;
         }
         this.showConfigureIntelliSenseButton = show;
-        if (client) {
+        if (client !== undefined) {
             client.setShowConfigureIntelliSenseButton(show);
         }
         if (show) {
```

Comment:
Do we still need the ` telemetry.logLanguageServerEvent('configureIntelliSenseStatusBar');` telemetry below?

Reply:
It's not the subject of this PR. We can discuss offline and add an issue if so.

Reply:
Yeah, but this PR adds telemetry to the same function that seems like it could be duplicate?

Reply:
...oh, looks like the telemetry doesn't track the same thing.